LanguageFeatures/Patterns/matching_map_A02_t03.dart

```diff
@@ -94,10 +94,10 @@ main() {
   Expect.throws(() {
     var <String, int?>{"key1": v1, "key2": v2} = map;
   });
-  Expect.equals("[key1];containsKey(key1);", map.log);
+  Expect.equals("[key1];containsKey(key1);[key2];containsKey(key2);", map.log);
```

Comment:
Does it do this? I would expect the invocation of `["key2"]` to yield a non-null value, and we would then _not_ invoke `containsKey("key2")`.

Reply:
`map` here is
```dart
final map = MyMap<String, int?>({"key1": null, "keyX": 42});
```
So `[key2]` should return `null`. But value type of the `map` is nullable (`int?`) so I'm expecting call of `containsKey(key2)` after `[key2]`

Reply:
Oh, it was "keyX", I missed that. Thanks!